### PR TITLE
feat(Timeline): migrate to tailwind

### PR DESCRIPTION
**Pull request by vbulant**
Closes [FEPLT-1702](https://kiwicom.atlassian.net/browse/FEPLT-1702)

# This Pull Request meets the following criteria:

### For Tailwind migrations:

- [x] Visual regression test suite has been added if does not exist.
- [x] `tailwind-migration-status.yaml` file has been updated with the migrated component(s).

### For new components:

- [ ] Tests have been added/adjusted for my new feature
- [ ] New Components are registered in index.js of my project
- [ ] New Components have `d.ts` files and are exported in `index.d.ts`

[FEPLT-1702]: https://kiwicom.atlassian.net/browse/FEPLT-1702?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/orbit-components/playwright/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
```
Comment (vbulant) [inline packages/orbit-components/playwright/index.html:2]: Needed for `ltr:` tailwind classes to work.

File: packages/orbit-components/src/Timeline/TimelineStep/components/TypeIcon.tsx [hunk 2]
```diff
@@ -31,11 +31,40 @@ const getTypeIcon = ({ type, active }: { type?: Type; active: boolean }) => {
...
+  mobile?: boolean;
+}
+
+const TypeIcon = ({ type, active, mobile }: Props) => {
   return (
-    <StyledIconWrapper mobile={mobile} status={type} active={active}>
+    <div
+      className={cx(
+        "w-md z-default relative flex h-[20px] shrink-0 items-center justify-center",
+        mobile && "mx-xxs",
```
[the remark this answers is not in the document]
Comment (vbulant) [reply packages/orbit-components/src/Timeline/TimelineStep/components/TypeIcon.tsx]: I think it seemed weird to me to have `min-width` set when the icon always had the same fixed width. But don’t remember all the nuances. Changed it as you suggested ✅

File: packages/orbit-components/src/Timeline/Timeline.ct-story.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        <TimelineStep type="critical" label="Waiting for the carrier" subLabel="5th May 14:04">
+          We’ll wait for the carrier(s) to send us the refund and contact them again if necessary.
+        </TimelineStep>
+        <TimelineStep type="info" label="Carrier is refunding" subLabel="6th May 14:04">
+          The carrier has sent us a refund. There might be more depending on their policy.
+        </TimelineStep>
+        <TimelineStep label="Refunded" subLabel="7th May 10:25">
+          We’ll forward you all refunds from the carrier(s) after we receive it.
+        </TimelineStep>
+      </Timeline>
```
[the remark this answers is not in the document]
Comment (vbulant) [reply packages/orbit-components/src/Timeline/Timeline.ct-story.tsx]: > Couldn't this example be inside the DIRECTIONS array map, so we don't repeat it for the direction="row"?

Good point! Updated

> I'd even suggest two more steps: only children (no sublabel) & only label (no children nor sublabel).

Label is a required prop. But I’ve added a dedicated instance with both directions and two steps: label only and label + children.

[the remark this answers is not in the document]
Comment (DSil) [reply packages/orbit-components/src/Timeline/Timeline.ct-story.tsx]: I meant sublabel, not label. So three options:
- Only label
- Label + sublabel (this one is missing)
- Label + children
- Label + children + sublabel

An example of the missing one is:
image

[the remark this answers is not in the document]
Comment (vbulant) [reply packages/orbit-components/src/Timeline/Timeline.ct-story.tsx]: Got you. It should be good now 👍

**Review by DSil — APPROVED** [review]

File: packages/orbit-components/src/Timeline/Timeline.ct-story.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (DSil) [reply packages/orbit-components/src/Timeline/Timeline.ct-story.tsx]: Perfect! 👌 thank you!

**Pull request merged**